pkg/jx/cmd/delete_app.go

```diff
@@ -97,16 +99,76 @@ func NewCmdDeleteApp(f Factory, in terminal.FileReader, out terminal.FileWriter,
 
 // Run implements this command
 func (o *DeleteAppOptions) Run() error {
+	var err error
+	var ns string
+	o.jxClient, ns, err = o.Factory.CreateJXClient()
+	o.currentNamespace = ns
+	if err != nil {
+		return errors.Wrap(err, "getting jx client")
+	}
+	isProw, err := o.isProw()
+	if err != nil {
+		return err
+	}
+	var deletedApplications []string
+	if isProw {
+		deletedApplications, err = o.deleteProwApp()
+	} else {
+		deletedApplications, err = o.deleteJenkinsApp()
+	}
+
+	if err != nil {
+		return errors.Wrapf(err, "deleting application")
+	}
+	log.Infof("Deleted Application(s): %s\n", util.ColorInfo(strings.Join(deletedApplications, ",")))
+	return nil
+}
+
+func (o *DeleteAppOptions) deleteProwApp() (deletedApplications []string, err error) {
+	envMap, _, err := kube.GetOrderedEnvironments(o.jxClient, "")
+	currentUser, err := user.Current()
+	if err != nil {
+		return deletedApplications, errors.Wrap(err, "getting current user")
+	}
+
+	kubeClient, ns, err := o.Factory.CreateClient()
+	if err != nil {
+		return deletedApplications, errors.Wrap(err, "getting kube client")
+	}
+
+	repos := make([]string, len(o.Args))
+	for i, repo := range o.Args {
+		// FIXME - get the repo name (ie, org/repo rather than just repo)
+		repos[i] = "cb-kubecd/" + repo
```

Comment:
🐛 You will need to get the org name from somewhere and not hard-code it here. I think it's a new option/flag on the `jx delete application` command (probably `--org`)